image/Makefile

```diff
@@ -0,0 +1,101 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
On #314 I suggest the file being rename. Apart from that, I would include a comment about its purpose (i.e. it should be included by other Makefiles).